README.md

```diff
@@ -76,7 +77,6 @@ $ scss-bundle -c scss-bundle.config.json
 | --includePaths             | includePaths         | array          | Include paths for resolving imports                               |                       |         |
 | --project, -p              | project              | string         | Project location, where `node_modules` are located.               |                       |         |
 | --ignoredImports           | ignoredImports       | array          | Ignore resolving import content by matching a regular expression. |                       |         |
-| --watch, -w                | watch                | boolean        | Watch files for changes.                                          |                       |         |

```

Comment:
Removed this duplicate & incorrect --watch argument reference that appears to have snuck in a while back. Pretty sure this wasn't intended to ever make it into the readme.